src/main/connector/hardware/LinuxIpmiTool/LinuxIpmiTool.yaml

```diff
@@ -0,0 +1,234 @@
+---
+extends:
+- ../../semconv/Hardware
+- ../../semconv/System
+connector:
+  displayName: IPMI
```

Comment:
We should rename the displayName to `Linux IPMI (ipmitool)`. And rename [ IPMITool connnector ](https://github.com/MetricsHub/community-connectors/blob/ec98e48fb0643cc9df0f7dd9188183d73fcb5dc9/src/main/connector/hardware/IpmiTool/IpmiTool.yaml) displayName to `IPMI Over-LAN`